widgets/features/features.php

```diff
@@ -253,6 +253,7 @@ function get_less_variables( $instance ) {
 			}
 		}
 
+		$less_vars['responsive_breakpoint'] = siteorigin_panels_setting('mobile-width') . 'px';
```

Comment:
Something along the lines of 

`$less_vars['responsive_breakpoint'] = ( function_exists( 'siteorigin_panels_setting' ) ? siteorigin_panels_setting('mobile-width') : 520 ) . 'px';`